<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegionGrowingSegmentationWidget</class>
 <widget class="QWidget" name="RegionGrowingSegmentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Segments</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>172</width>
           <height>174</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QListWidget" name="segmentsListWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newSegmentButton">
        <property name="text">
         <string>New Segment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeSegmentButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="seed_point_groupbox">
     <property name="title">
      <string>Seed Points</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QScrollArea" name="scrollArea_3">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>172</width>
           <height>174</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QListWidget" name="seedsListWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newSeedButton">
        <property name="text">
         <string>New Seed Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeSeedButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>Segmentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Tolerance</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QDoubleSpinBox" name="toleranceSpinbox">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="performSegmentationButton">
        <property name="text">
         <string>Perform</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveParameterButton">
        <property name="text">
         <string>Save Parameter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="load_ParameterButton">
        <property name="text">
         <string>Load Parameter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
